modules/logging-forwarding-azure.adoc

```diff
@@ -128,10 +128,10 @@ spec:
        name: my-secret
   pipelines:
    - name: app-pipeline
```

Comment:
can you align '-' with the '-' in line 120. Remove one pre-space.

Reply:
Hello @anpingli ,
Thank you very much for this suggestion.
Line number 120 is correct.

I performed the suggested changes in line number 130, removed one `pre-space`

Request you to please have a look.

Regards,
Prithviraj Patil